client/src/containers/NotRegisteredPage.tsx

```diff
@@ -289,10 +301,11 @@ export default class NotRegisteredPage extends Component {
                   <Trans>Share this page with your neighbors</Trans>
                 </p>
                 <SocialShare
-                  location="nycha-page"
+                  location="not-registered-page"
```

Comment:
Interesting, is there a reason this string is changing from `nycha-page` to `not-registered-page`?

Reply:
Yes, this is used for utm reporting in our social share— this should appropriately be not-registered to be a better label for this page.